Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -23,7 +23,10 @@ @interface OWSMessageBubbleView () <OWSQuotedMessageViewDelegate, OWSContactShar
 
 @property (nonatomic) OWSBubbleView *bubbleView;
 
-@property (nonatomic) OWSBubbleShapeView *mediaShadowView;
+// TODO: We may only end up using a single shadow.
+@property (nonatomic) OWSBubbleShapeView *mediaShadowView1;
```

Comment:
Can you leave a comment explaining what the difference is between these shadows? 

It seems like you're trying to create a sort of non-linear effect by overlapping the two shadows?

Would innerShadow and outerShadow be accurate names?

Reply:
"Inner shadows" is a term (I think it originated with photoshop?) for shadows _on_ an object (as opposed to outside it) that highlight its edges, sort of like bas-relief.  

Anyway, the design has changed and we're not doing inner shadows any more.  I don't understand why we have two separate shadows - please ask Myles.